src/libs/auth.ts

```diff
@@ -55,16 +55,62 @@ const getAuthInstance = () => {
   return authStore.get().authContext;
 };
 
-export const signOut = () => {
-  // TODO: invalidate runtime cookies https://broadworkbench.atlassian.net/browse/IA-3498
+export const enum SignOutCauses {
```

Comment:
Generally, union types are preferable over enums.
https://www.bam.tech/en/article/should-you-use-enums-or-union-types-in-typescript
https://docs.google.com/document/d/17jMMUyBABc6HFCsbfj6MURWHqpFxWC1f0zielaiw73s/edit

```suggestion
export const SignOutCause =
  | 'requested'
  | 'disabled'
  | 'declinedTos'
  ...
```